starfish/pipeline/features/spots/detector/__init__.py

```diff
@@ -43,6 +51,11 @@ def _cli(cls, args, print_help=False):
 
         print('Detecting Spots ...')
         image_stack = ImageStack.from_path_or_url(args.input)
+        if args.blobs_stack is not None:
```

Comment:
i think this breaks the gaussian spot detector of blobs_stack is none. according to the typing, the find method expects an xarray or ndarray